cparser/processor.js

```diff
@@ -53,6 +61,10 @@ module.exports = that = {
 
 			var includeStr = '#include "application.h"';
 			var appDotHInclude = fileBuffer.indexOf(includeStr);
+			if (appDotHInclude === -1) {
```

Comment:
Another way to get application.h is via Arduino.h all by itself, which includes Particle.h which includes application.h.